torchvision/prototype/transforms/_augment.py

```diff
@@ -190,6 +190,90 @@ def _transform(self, inpt: Any, params: Dict[str, Any]) -> Any:
             return inpt
 
 
+def flatten_and_extract_data(
+    inputs: Any, **target_types_or_checks: Tuple[Union[Type, Callable[[Any], bool]], ...]
+) -> Tuple[Tuple[List[Any], TreeSpec, List[Dict[str, int]]], List[features.TensorImageType], List[Dict[str, Any]]]:
+    # Images are special in the sense that they will always be extracted and returned
+    # separately. Internally however, they behave just as the other features.
+    types_or_checks: Dict[str, Tuple[Union[Type, Callable[[Any], bool]], ...]] = {
+        "images": (features.Image, PIL.Image.Image, features.is_simple_tensor),
```

Comment:
Should we avoid defaults and just explicitly pass images in the 2 cases? This might be useful if on the future we extended this for video. No strong opinions.